src/Contract/Operation/Spanable.php

```diff
@@ -18,7 +19,9 @@
 interface Spanable
 {
     /**
-     * @return Collection<TKey, T>
+     * @param callable(T, TKey, Iterator<TKey, T>): bool $callback
```

Comment:
shouldn't `Span` also accept variadic callable?

Reply:
Nice catch, indeed it could be!

Reply:
But it could be tackled in a new PR.